src/settings.py

```diff
@@ -0,0 +1,14 @@
+from pydantic.v1 import BaseSettings
+
+
+class Settings(BaseSettings):
+    PROJECT_NAME: str = "City Temperature Management API"
+
+    DATABASE_URL: str | None =  "sqlite+aiosqlite:///./city_weather.db"
```

Comment:
Consider using `Optional[str]` instead of `str | None` for the `DATABASE_URL` type hint to ensure compatibility with Python versions prior to 3.10.